card/mod/06_bootstrap/set/all/bootstrap/header.rb

```diff
@@ -1,11 +1,13 @@
 format :html do 
   view :header do |args|
     %{
-      <h3 class="card-header panel-title">
-        #{ _optional_render :toggle, args, :hide }
-        #{ _optional_render :title, args }
+      <div class="card-header panel-heading">
+        <h3 class="card-header-title panel-title" style="display:inline-block">
+          #{ _optional_render :toggle, args, :hide }
+          #{ _optional_render :title, args }
+        </h3>
         #{ _optional_render :menu, args }
-      </h3>
+      </div>
     }
```

Comment:
header slot has panel-heading class
title is wrapped with new h3 tag. I think this is right to wrap it, because
1. bootstrap expects hx tag
2. title view sometimes returns span, sometime anchor, sometimes anchor in span...

it's nice that the new arrangment won't break CSS that looks like .card-frame > .card-header (I think there's a lot of that?)

BOO for inline styling (needs fixing!)